addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -372,12 +372,16 @@
         }
     }
 
-    .o_data_row.o_selected_row > .o_data_cell.o_required_modifier:not(.o_readonly_modifier) {
-        border-bottom: 1px solid $o-input-border-required;
+    .o_data_row.o_selected_row > .o_data_cell.o_required_modifier {
+        &:not(:has(.o_readonly_modifier)) {
```

Comment:
No need to split in 2 lines. You can keep it like it was before your very last push.

https://github.com/odoo/odoo/compare/6eabe42a72e1f45a91c3e22f45493de170e2c8e5..a0a858ba5a4ed6b8b1c7c457da4fc09b901ece2f